gen/dcos-config.yaml

```diff
@@ -447,8 +447,6 @@ package:
       {
         "PATH": "/usr/bin:/bin",
         "SHELL": "/usr/bin/bash",
-        "LD_LIBRARY_PATH": "/opt/mesosphere/lib",
```

Comment:
@adam-mesos The need to remove this has been stated to the frameworks teams repeatedly since DC/OS 1.3. That is a lot more than 6 months that they've had notice to fix it. Giving more time I don't think is going to result in a solution.